spec/models/spotlight/reindexing_log_entry_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe Spotlight::ReindexingLogEntry, type: :model do
+  subject { FactoryGirl.build(:reindexing_log_entry) }
+
+  describe 'scope' do
+    before(:all) do
+      # we only want to persist these rows for the duration a given test run...
+      DatabaseCleaner.start
```

Comment:
Hmm, was this added because of some odd behavior you were seeing?  I would think this would be not necessary due to https://github.com/projectblacklight/spotlight/blob/master/spec/spec_helper.rb#L56-L70.